examples/misc/persistent-chat-meilisearch.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+use Symfony\AI\Agent\Agent;
+use Symfony\AI\Agent\Bridge\Meilisearch\MessageStore;
+use Symfony\AI\Agent\Chat;
+use Symfony\AI\Platform\Bridge\OpenAi\Gpt;
+use Symfony\AI\Platform\Bridge\OpenAi\PlatformFactory;
+use Symfony\AI\Platform\Message\Message;
+use Symfony\AI\Platform\Message\MessageBag;
+
+require_once dirname(__DIR__).'/bootstrap.php';
+
+$platform = PlatformFactory::create(env('OPENAI_API_KEY'), http_client());
+$llm = new Gpt(Gpt::GPT_4O_MINI);
+
+$agent = new Agent($platform, $llm, logger: logger());
+$store = new MessageStore(
+    http_client(),
+    env('MEILISEARCH_HOST'),
+    env('MEILISEARCH_API_KEY'),
+    'chat',
+);
+$store->initialize();
+
+$chat = new Chat($agent, $store);
```

Comment:
What about letting the Chat call initialize internally if it is some kind of initalizeable store?

Just thinking out loud